site_settings/models.py

```diff
@@ -4,6 +4,17 @@
 from wagtail.contrib.settings.models import BaseSetting, register_setting
 
 
+
+@register_setting
+class SocialMediaSettings(BaseSetting):
+
+    social_media_header = models.CharField(max_length=255, default='Other ways to stalk me')
```

Comment:
I think this could be more generic such as `default='Follow Me'`

Reply:
Fixed